pkgs/applications/terminal-emulators/gnome-console/default.nix

```diff
@@ -51,6 +52,11 @@ stdenv.mkDerivation rec {
     wrapGAppsHook
   ];
 
+  mesonFlags = lib.optionals (!stdenv.isLinux) [
+    "-Dnautilus=disabled"
```

Comment:
Werror should always be passed

Reply:
Fetched [upstream patch](https://gitlab.gnome.org/GNOME/console/-/commit/0e29a417d52e27da62f5cac461400be6a764dc65).